### DISCUSSION TITLE
Support `pscale auth login` from a headless terminal

### DISCUSSION THREAD

**Discussion by enisoc** [planetscale/discussion#11] [Feature Requests] [answered]
I would like to run commands like `pscale shell` from a headless jump box (e.g. an EC2 instance). However, when I try to `pscale auth login` on that box I get this:

```console
$ wget https://github.com/planetscale/cli/releases/download/v0.32.0/pscale_0.32.0_linux_amd64.tar.gz
$ tar -zxf pscale_0.32.0_linux_amd64.tar.gz
$ ./pscale auth login
Press Enter to authenticate via your browser...

Error: error opening browser: exec: "xdg-open": executable file not found in $PATH
```

**Comment by enisoc** [comment]
I was able to work around this by creating a no-op script called `xdg-open` in my PATH. After that, `pscale auth login` (in the remote terminal) prints a link that I can load in my local browser.

**Comment by lkburns** [comment]
Thanks for catching this! We're working on a fix.

**Comment by fatih** [comment]
Thank you @enisoc for the feedback. I improved the flow with this PR: https://github.com/planetscale/cli/pull/248 It'll be available in the next release of the CLI.

**Comment by aquarapid** [comment, marked as answer]
Just to close the loop;  since @fatih fixed this, the interaction using a newer version of `pscale` is now something like:

```
$ pscale auth login
Failed to open a browser: exec: "xdg-open": executable file not found in $PATH

Confirmation Code: XXXXXXXX

If something goes wrong, copy and paste this URL into your browser: https://auth.planetscale.com/oauth/device?user_code=XXXXXXXX

⠙ Waiting for confirmation... 
```
